src/Knp/Snappy/Pdf.php

```diff
@@ -29,19 +32,15 @@ public function __construct($binary = null, array $options = array(), array $env
      */
     protected function handleOptions(array $options = array())
     {
-        $this->hasHtmlHeader = false;
-        $this->hasHtmlFooter = false;
-        
-        if ($this->isFileHeader($options) && !$this->isFile($options['header-html'])) {
```

Comment:
it is handled [here](https://github.com/KnpLabs/snappy/pull/160/files#diff-3e2f6acd9d1c2cb5db8e49bdde73d864R40)